samples/Hardware.GiantGecko/GiantGecko.ReadDeviceIDs/category.txt

```diff
@@ -0,0 +1 @@
+stm32
```

Comment:
is the GiantGecko an STM32 based? If not, then worth creating a ne category and adjusting the build tool

Reply:
Yes. That was added and it's listed properly.